images/tidb-backup-manager/entrypoint.sh

```diff
@@ -19,7 +19,7 @@ echo "Create rclone.conf file."
 cat <<EOF > /tmp/rclone.conf
 [s3]
 type = s3
-env_auth = false
+env_auth = true
```

Comment:
We also have to consider the non-IAM role case.

Reply:
https://rclone.org/s3/#authentication the secret key and access key is from env AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY, If  access_key and secret_access_key is set, the env have this two parameters. set it to true is a save way for IAM and non-IAM.